.eslintrc

```diff
@@ -14,9 +14,10 @@
   },
   "parser": "babel-eslint",
   "rules": {
-    "object-curly-spacing": 0,
-    "space-before-function-paren": [2, "never"],
-    "react/prefer-stateless-function": 0,
+    "arrow-body-style": "off",
```

Comment:
For the object case you need to wrap it in parens. It seems a little strange but I don't think it looks too bad.

```js
const getConfig = () => ({
  some: 'config',
});